pinot-common/src/main/java/org/apache/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -376,6 +385,7 @@ public static boolean removeSegmentZKMetadata(ZkHelixPropertyStore<ZNRecord> pro
     return propertyStore.remove(constructPropertyStorePathForSegment(tableNameWithType, segmentName),
         AccessOption.PERSISTENT);
   }
+
```

Comment:
nit: whitespace

Reply:
This is no new line between the two method. So this is alright.